### ISSUE TITLE
Discussion about licenses

### ISSUE THREAD

**Issue by b-rodrigues** [rbind/b-rodrigues.github.com#4]

**Comment by fkohrt** [comment]
> Free software is like open source software, but it’s much more restrictive.
> [...]
> [Free software] does not allow is [...] this: I cannot distribute (by selling or for free) the program without its source code. 

I am not so sure about that.

1. Software licensed under MIT or BSD license is also free software (according to [FSF's definition](https://www.gnu.org/philosophy/free-sw.html).
2. Software licensed under GPL is also open source software (according to [OSI's definition](https://opensource.org/osd)).

The actual difference between free and open source software is much more nuanced (if existent at all):

[It is expected](https://www.gnu.org/philosophy/open-source-misses-the-point.html) by the FSF that all free software should also considered OSS by the OSI. For the other way round, there may be cases where this is not true, but _most of the OSS should be free software [as well](https://www.gnu.org/philosophy/free-open-overlap.html)_. If requiring free software over OSS is not so much about the amount of software matching that criteria, what else is it about?

Free software is about protecting the user's freedom. From this perspective, nonfree software is a social problem. OSS instead has a more pragmatic approach, it is centered around the product being better in the end. Nonfree software just cuts efficiency on that road. This makes clear that requiring free software instead of OSS is about spreading freedom as a value.

**Comment by b-rodrigues** [comment]
Thank you for your comment; but I do think that it would be impossible to distribute (or sell) GPL'ed software without its source code.

**Comment by bokov** [comment]
The

> Thank you for your comment; but I do think that it would be impossible to distribute (or sell) GPL'ed software without its source code.

The OP's point is that you are incorrectly using terminology in a way makes your article spread confusion. To me it sounds like your article is really about restrictive vs n ok non-restrictive open source licenses

**Comment by fkohrt** [comment]
> I do think that it would be impossible to distribute (or sell) GPL'ed software without its source code.

I agree. But whether distribution of source code is required for derivative works is a question of the specific license terms and not one of OSD-conformance vs. FSF-freedom.

I agree with @bokov that your article – which I do find valuable as it discusses licensing in the R community 😀 – concerns the distinction between permissive vs. restrictive licenses, but not open source vs. free.

**Comment by b-rodrigues** [comment]
I see, I might clarify the post now, thank you for your inputs @bokov and @fkohrt !
